art.taunoerik.tauno-serial-plotter.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Tauno Erik <sedumacre@email.com> -->
+<component type="desktop-application">
+  <id>art.taunoerik.tauno-serial-plotter</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <name>Tauno Serial Plotter</name>
+  <summary>Simple serial plotter for Arduino and others similar devices.</summary>
+
+  <description>
+    <p>
+      Tauno-Serial-Plotter is simple serial plotter for Arduino and similar others.
+    </p>
+    
+    <p>Features:</p>
+    <ul>
+      <li>Simple user interface</li>
+      <li>Plotting of multiple variables, with different colors for each</li>
+      <li>Can plot both integers and floats</li>
+      <li>Can plot negative values</li>
+      <li>Auto-scrolls the Time scale (X axis)</li>
+      <li>Auto-resizes the Data scale (Y axis)</li>
+    </ul>
+    
+    <p>
+      Incoming serial data should be string. Ending with new line character. Numbers (int and float) can be separated with almost any character. Like: "label2la15be17el28/31/42/54 78\n" or "a2b1.5c1.7d2.8/3.1/4.2/5.4 7.8\n". But not with - unless it is a negative number: "-10\"
+    </p>
+  </description>
+
+  <launchable type="desktop-id">art.taunoerik.tauno-serial-plotter.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>The main window</caption>
+      <image>https://raw.githubusercontent.com/taunoe/tauno-serial-plotter/master/img/screenshot.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Exporting the data</caption>
+      <image>https://raw.githubusercontent.com/taunoe/tauno-serial-plotter/master/img/graph-settings.png</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://github.com/taunoe/tauno-serial-plotter</url>
+
+
+
+  <releases>
+    <release version="1.17" date="2022-01-07">
+      <description>
+        <p>First Flatpak release. And fixes some minor.</p>
```

Comment:
this should be about the app, not the flatpak.

Also incidentally this should be part of a future release: it's not specific to Flatpak.